apps/desktop/desktop/src/main/resources/css/bisq_easy.css

```
@@ -1307,6 +1313,7 @@
     -fx-fill: -bisq-white;
     -fx-stroke: -bisq-black;
     -fx-stroke-width: 1.5;
+    -fx-cursor: "hand";
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)